framework/test/bundles/libRWithAppendedResources/CMakeLists.txt

```diff
@@ -12,3 +12,6 @@ usFunctionCreateTestBundleWithResources(TestBundleRA
   APPEND_RESOURCES
 )
 
+if(APPLE)
+	set_target_properties(TestBundleRA PROPERTIES XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+endif()
```

Comment:
Is it possible to generalize this fix such that any bundle which is built on macOS using the resource compiler's "append" mode will have this xcode attribute set?

Reply:
Added logic to "usFunctionEmbedResources.cmake" file instead, to generically disable code-signing for any bundle appending resources on macOS.